src/api/repos/business-repo.ts

```diff
@@ -131,6 +131,7 @@ const addNewEmployee = async (id: string, employee: Employee) => {
 const addNewActivity = async (id: string, activity: Activity, address: Address) => {
     await Business.sync({ alter: true });
     await Activity.sync({ alter: true });
+    await Address.sync({ alter: true });
```

Comment:
Honestly, I'm not sure about creating an Address table. In the mobile app, I am using mapbox to suggest addresses to users as they are typing so I'm expecting the addresses to be consistent. I think we can store them in a single column in the Activity table

Reply:
This is because I'm reusing the Activity and Address tables from activity creation for Business. If I make the change to not use a separate Address table for Activity's Address then it'll apply for both Business and User.

I don't think it's a bad idea to remove the use of the Address table though. Had it agreed with Philippe before for the Business, that's why I'm sticking with it. 

What do you think?

Reply:
An entire table for addresses is a bit overkill in my opinion, especially since we really won't use them for anything really important except to show users where the activity will take place. I thought adding address suggestions/autofill is enough and provides a good user experience since the user doesn't have to type everything out and, in return we get properly formatted mapbox addresses